app/views/games/show.html.erb

```diff
@@ -1,14 +1,16 @@
 <br />
-<h1 class="text-center title"><%= @game.name %></h1>
+<h1 class="text-center title"><%= current_game.name %></h1>
 <br />
 <table class="chess_table">
   <% 7.downto(0).each do |row_number| %>
     <tr>
       <% 0.upto(7).each do |column_number| %>
         <td class="<%= (row_number + column_number).even? ? "black_space" : "white_space"%>">
-          <% piece = @game.pieces.find_by(x_position: column_number, y_position: row_number) %>
+          <% piece = current_game.pieces.find_by(x_position: column_number, y_position: row_number) %>
           <% if piece %>
-            <p class='piece'><%= piece.to_unicode %></p>
+            <%= link_to game_piece_path(current_game.pieces) do %>
```

Comment:
Try changing this to `game_piece_path(game, piece)` (I think that's correct).

In a console, if you run `rake routes`. You'll be able to see all of the routes and what their `helper` is. So you should see a `game_piece_path` there. Seeing that can sometimes be helpful in figuring out which path helper to use (all of the `something_path` methods are considered a "path helper").

😄